sentry-android-core/build.gradle.kts

```diff
@@ -79,6 +79,8 @@ dependencies {
     // lifecycle processor, session tracking
     implementation(Config.Libs.lifecycleProcess)
     implementation(Config.Libs.lifecycleCommonJava8)
+    // core is a transitive dependency of appcompat btw
+    implementation(Config.Libs.androidxCore)
```

Comment:
androidx core is a transitive dependency of the appcompat package which contains the `AppCompatActivity` so I believe almost every app already has this dependency anyway

Reply:
either we keep it as it is and deal with dependency conflicts or we create a new package and tell people to add it, wdyt?

Reply:
The last release for the Android Support Library was in September 2018 with [28.0.0](https://developer.android.com/topic/libraries/support-library/revisions#28-0-0). It doesn't receive any bug fixes anymore. So you have to migrate to AndroidX. I think it is fine to pull in that dependency.